xterm/index.cgi

```diff
@@ -7,13 +7,72 @@
 # XXX Virtualmin integration?
 
 require './xterm-lib.pl';
+&ReadParse();
 
+# Get Webmin current version for links serial
+my $wver = &get_webmin_version();
+$wver =~ s/\.//;
+
+# Build Xterm dependency links
+my $termlinks = 
+	{ 'css' => ['xterm.css?$wver'],
```

Comment:
Why include the version here?

Reply:
This is to tell the browser, not to use cached CSS and JS files, which potentially can change with each new Webmin version.

Reply:
Ok, makes sense!